unit-tests/README.md

```diff
@@ -0,0 +1,268 @@
+# DTS scripts unit tests
+
+This directory contains stub scripts for testing DTS update and deploy logic
+in complex scenarios.
+
+##  Running on host
+
+Running test on the host may result in unpredictable results because of the
+missing programs and different version of certain tools. It is advised to run
+the DTS image in QEMU as a development environment. Running on host is
+generally not supported and should be avoided.
+
+## Running in QEMU
+
+### Credentials setup
+
+We need credentials for each test variant. You can use provided template and
+fill it in accordingly.
+
+```bash
+cp des-credentials.sh.example des-credentials.sh
+```
+
+### Running automatically
+
+Some scenarios are have been already migrated into [OSFV](TBD).
```

Comment:
Currently this 'link' doesn't lead anywhere. Should it stay that way?

Reply:
I think I meant to use this MR back then: https://github.com/Dasharo/open-source-firmware-validation/pull/333